code/mathematical-algorithms/Check_is_square/check_is_square.swift

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env swift
+// Part of Cosmos by OpenGenus Foundation
+import Darwin
+
+func findingSquare(num : Int) -> Int {
```

Comment:
It would be better to make the function return a `Bool` as it checks whether it is a square or not.